lib/charms/operator_libs_linux/v0/apt.py

```diff
@@ -206,13 +206,9 @@ def __eq__(self, other) -> bool:
         Returns:
           A boolean reflecting equality
         """
-        return (
-            isinstance(other, self.__class__)
-            and (
-                self._name,
-                self._version.number,
-            )
-            == (other._name, other._version.number)
+        return isinstance(other, self.__class__) and (self._name, self._version.number) == (
+            other._name,
+            other._version.number,
```

Comment:
Most of the changes in this PR seem to be formatting changes -- one to be aware of in future as it makes seeing the actual changes a little harder.

Reply:
Ah I see, my bad, I ran black on this and it reformatted the file. Should I revert these changes?

Reply:
Probably best just keep this PR clean. The code should have already been formatted through `tox -e lint` (which uses `black` with a line length of 99)